juniper_official/system/check-system-cpu-memory.rule

```diff
@@ -187,7 +187,7 @@ healthbot {
                      then {
                          status {
                              color red;
-                             message "$routing-engine CPU utilization($re-cpu-utilization) exceeds high threshold($re-cpu-utilization-high-threshold)";
+                             message "$routing-engine CPU utilization($re-cpu-utilization) exceeds critical threshold($re-cpu-utilization-high-threshold)";
```

Comment:
Below when conditions needs to be changed. 'all' should be changed to any

`                 term is-re-cpu-utilization-middle {
                     when {
                         greater-than-or-equal-to "$re-cpu-utilization" "$re-cpu-utilization-low-threshold" {
                             time-range 3offset;
                             all;
                         }
                         less-than "$re-cpu-utilization" "$re-cpu-utilization-high-threshold" {
                             time-range 3offset;
                         }						 
                     }`